examples/policies/timed-mint-policy/timed-mint-policy-v1.pact

```diff
@@ -0,0 +1,134 @@
+(namespace (read-msg 'ns))
+
+(module timed-mint-policy-v1 GOVERNANCE
+
+  @doc "Policy for timed-mint tokens with royalty and quoted sale in coin."
+
+  (defconst ADMIN-KS:string "marmalade-v2.marmalade-contract-admin")
+
+  (defcap GOVERNANCE ()
+    (enforce-guard ADMIN-KS))
+
+  (implements kip.token-policy-v2)
+  (use kip.token-policy-v2 [token-info])
+  (use marmalade-v2.policy-manager)
+
+  (defschema timed-mint-schema
+    max-supply:decimal
+    mint-start-time:time
```

Comment:
We're using timestamps in general, so it'd be nice to take in times as integers, in timestamps. Users can use `to-timestamp` function if they'd like to pass in time in time format.